scripts/zones/Port_Jeuno/npcs/Shemo.lua

```diff
@@ -0,0 +1,67 @@
+-----------------------------------
+-- Area: Port Jeuno
+--  NPC: Shemo
+-----------------------------------
+local ID = require("scripts/zones/Port_Jeuno/IDs")
+require("scripts/globals/settings")
+require("scripts/globals/quests")
+require("scripts/globals/utils")
+-----------------------------------
+local entity = {}
+
+local beastman_seal = 0         -- �b�l���
+local kindred_seal = 1          -- �b�_���
+local kindred_crest = 2         -- ���l���
+local high_kindred_crest = 3    -- �������
+local sacred_kindred_crest = 4  -- ���_���
```

Comment:
This comments were originaly in Japanesse, I think, but they have lost format. Could you please change them to english or remove them?

Reply:
It has been deleted